catalyst-gateway/bin/src/db/index/queries/purge/mod.rs

```diff
@@ -0,0 +1,238 @@
+//! Queries for purging volatile data.
+
+pub(crate) mod txo_ada;
+
+use std::{fmt::Debug, sync::Arc};
+
+use scylla::{
+    prepared_statement::PreparedStatement, serialize::row::SerializeRow,
+    transport::iterator::RowIterator, Session,
+};
+
+use super::{FallibleQueryResults, SizedBatch};
+use crate::settings::cassandra_db;
+
+/// No parameters
+const NO_PARAMS: () = ();
+
+/// All prepared DELETE query statements (purge DB table rows).
+#[derive(strum_macros::Display)]
+#[allow(dead_code)]
```

Comment:
Is it needed here ?

Reply:
It's temporary, I'm incorporating functionality, so this is WIP.